onadata/apps/logger/models/xform.py

```diff
@@ -870,6 +870,15 @@ def save(self, *args, **kwargs):
         if 'skip_xls_read' in kwargs:
             del kwargs['skip_xls_read']
 
+        if (self.id_string and len(
+            self.id_string) > self.MAX_ID_LENGTH) or \
+            (self.sms_id_string and len(
+                self.sms_id_string) > self.MAX_ID_LENGTH):
+            raise XLSFormError(
+                    _(u'The XForm id_string provided exceeds'
+                        ' the stipulated 100 characters.'
```

Comment:
I see we have a value for `MAX_ID_LENGTH` could we use it in the error message? I will propose:
```python
_(u'The XForm id_string provided exceeds %s characters. '
     'Please change the 'id_string' or "form_id" values in settings sheet or '
      'reduce the file name if you do not have a settings sheets.' % self.MAX_ID_LENGTH)
```

Reply:
Changed this to include the `MAX_ID_LENGTH` attribute